addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/handler/TrackerHandler.java

```diff
@@ -106,6 +115,7 @@ private void loadTrackerConfig() {
     @Override
     public void initialize() {
         updateThingChannels();
+        updateStatus(ThingStatus.UNKNOWN);
```

Comment:
But there is a @deprecated super that does the same. Is it an anti-pattern to set it ONLINE in intialize?

Reply:
Well, as discussed: In 99% of all handlers, you have to do a proper detection whether your remote device is working/reachable or not and set the status dynamically to ONLINE or OFFLINE. But in our case here, we came to the conclusion that we cannot do this and hence have to optimistically assume that it is always ONLINE.

Reply:
Yes. The TrackerHandler.updateLocation and doTransition updates the thing to ONLINE. I thought that after a restart we could wait for the first update record to arrive to set the thing to ONLINE.

Reply:
Yeah, but after a restart the user might be at home for 5 days and it will stay on UNKNOWN for all that time and when checking the system, this appears like a problem, while everything is fine. So I think ONLINE is the better choice.

Reply:
I updated my comment above while you were typing... Still OK?

Reply:
If it can still take 5 days until the status transitions from UNKOWN to ONLINE, I'd stick to my suggestion.

Reply:
OK